### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by clintonb**
Added prereqs (e.g. npm) and updated getting started instructions.

XCOM-367

FYI @jimabramson @rlucioni @awais786

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
 
     $ python manage.py oscar_populate_countries
 
-To stand up the development server, run::
+4. Run the development server::
 
     $ make serve
 
-By default, the Django Debug Toolbar is disabled. To enable it, set the ENABLE_DJANGO_TOOLBAR environment variable.
+By default the Django Default Toolbar is disabled. To enable it set the environmental variable ENABLE_DJANGO_TOOLBAR.
```
Comment (jimabramson) [inline README.rst]: s/environmental/environment/

Comment (jimabramson) [inline README.rst]: comma after "default"

Comment (jimabramson) [inline README.rst]: and after "to enable it"

Comment (clintonb) [inline README.rst]: Updated.

**Comment by jimabramson** [comment]
:+1:

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**